### DISCUSSION TITLE
卸载应用服务器卡死

### DISCUSSION THREAD

**Discussion by CV-Devlau** [1Panel-dev/1Panel#4213] [Q&A]
### 1Panel 版本

1.10.1

### 问题描述

服务器配置 1c2g
安装应用 Mysql、Halo
进行安装南墙waf服务器卡死，重启服务器后，安装雷池waf同样卡死，服务器重启需要几分钟
服务器监控显示 cpu平常使用在7%左右，安装waf时占用100%卡死，重启服务器后waf已经安装成功，cpu占用恢复，准备卸载halo，卸载同样卡死

单独使用docker 安装 halo、mysql 雷池waf 无问题可以运行。

猜测是不是多线程操作导致cpu卡死？

### 重现步骤

1c机器安装 Mysql、Halo 然后安装南墙waf

### 期待的正确结果

在单核服务器上，可以慢但应该能够正常使用，不出现卡死的情况

### 附加信息

![image](https://github.com/1Panel-dev/1Panel/assets/63354237/58e99e9a-9879-4bb8-9db5-f93d7fc71ed0)

**Comment by zhengkunwang223** [comment]
1Panel 本身也要消耗 cpu 和 内存
执行操作也会增加 cpu 和 内存 的占用
halo uuwaf 雷池 的占用也不低
1c2g 的机器确实比较吃力

**Comment by CV-Devlau** [comment]
> 1Panel 本身也要消耗 cpu 和 内存 执行操作也会增加 cpu 和 内存 的占用 halo uuwaf 雷池 的占用也不低 1c2g 的机器确实比较吃力

目前 1Panel  halo pg 雷池 运行cpu占用 在10%左右，我手动通过docker 安装没有问题，所以我觉得确实如你所说1panel执行操作也会增加cpu占用，并且对于1c来说还不少，直接使用docker安装 雷池时我看监控里cpu峰值利用率在70%。
通过1Panel操作卡死，1Panel至少得再占用30%我觉得这部分占用有点多了，我现在不清楚这期间1Panel干了什么事

**Comment by CV-Devlau** [comment]
可能是opencloudos 系统的原因，目前换成了centos 7.6 没有哦出现这个问题
